src/matlab2tikz.m

```diff
@@ -154,32 +154,21 @@ function matlab2tikz(varargin)
                             'Octave', struct('name','3.8', 'num',[3 8]));
     checkDeprecatedEnvironment(minimalVersion);
 
-    m2t.cmdOpts = [];
-    m2t.currentHandles = [];
-
-    m2t.transform = []; % For hgtransform groups
+    m2t.args            = []; % For command line arguments
+    m2t.current         = []; % For currently active objects
+    m2t.transform       = []; % For hgtransform groups
     m2t.pgfplotsVersion = [1,3];
```

Comment:
That line only specifies the minimal version that is needed. Also, there is nothing fancy going on with the version number: we dynamically determine which version is the minimal supported one depending on the feature we use and just show it to the user. In standalone, we always use the `newest` version.